src/content/tutorials/observability/multi-tenancy/_index.md

```diff
@@ -1,15 +1,201 @@
+
+### By service or application
+
+Create separation at the service level:
+
+```yaml
+tenants:
+- user-service
+- payment-service
+- notification-service
+- analytics-service
+```
+
+**Use case:** Microservices where each service team manages their own monitoring.
+
+### Hybrid approach
+
+Combine strategies using consistent naming:
+
+```yaml
+tenants:
+- prod-frontend
+- prod-backend
+- staging-frontend
+- staging-backend
+```
+
+**Use case:** Balance granular control with manageable complexity.
+
+## Tenant naming best practices
+
+Follow these guidelines when choosing tenant names:
+
+### Naming requirements
+
+Tenant names must follow [Grafana Mimir tenant ID restrictions](https://grafana.com/docs/mimir/latest/configure/about-tenant-ids/):
+
+- Alphanumeric characters (a-z, A-Z, 0-9) and special characters (!, -, _, ., *, ', (, ))
+- Between 1 and 150 characters
+- Can't use: `.`, `..`, `__mimir_cluster`
+
+### Recommended patterns
+
+**Environment-based:** `production`, `staging`, `development`
+
+**Team-based:** `platform-team`, `frontend-team`, `backend-team`
+
+**Service-based:** `user-service`, `payment-service`, `notification-service`
+
+**Hybrid:** `prod-frontend`, `stage-backend`, `dev-notifications`
+
+## Access control patterns
+
+### Organization-to-tenant mapping
+
+Map Grafana organizations to tenants based on your access needs:
+
+#### One-to-one mapping
+
+```yaml
+# Platform Team Organization
+tenants:
+- platform-team
+```
+
+**Best for:** Complete team isolation, clear ownership.
+
+#### One-to-many mapping
+
+```yaml
+# Engineering Organization
+tenants:
+- frontend-team
+- backend-team
+- platform-team
+```
+
+**Best for:** Cross-team collaboration, shared dashboards.
+
+### RBAC role assignment
+
+Assign roles based on user needs:
+
+```yaml
+rbac:
+  admins:
+  - customer:platform-admin      # Full access to organization
+  editors:
+  - customer:senior-engineer     # Can create/edit dashboards and alerts
+  viewers:
+  - customer:junior-engineer     # Read-only access to dashboards
+```
+
+## Data governance
+
+### Tenant governance
+
+The platform automatically enforces tenant governance to control data ingestion:
+
+- **Valid tenants:** Only tenants referenced in at least one `GrafanaOrganization` can receive data
+- **Data rejection:** Data sent to unlisted tenants gets dropped automatically
+- **Shared access:** Multiple organizations can reference the same tenant for shared datasets
+
+⚠️ **Important:** Removing a tenant from all organizations immediately stops data ingestion for that tenant.
+
+### Tenant lifecycle management
+
+**Creating tenants:**
+
+1. Define the tenant in `GrafanaOrganization` resource
+2. Configure data collection to use tenant label
+3. Verify data appears in organization dashboards
+
+**Removing tenants:**
+
+1. Stop sending data to the tenant
+2. Wait for retention period to expire
+3. Remove tenant from all `GrafanaOrganization` resources
```

Comment:
Same remark on having detailed steps either in a single page or linked pages.
We also can't ask use to just sit there and wait for the retention period to expires, we can at best explain that data will be permanently deleted once the retention period expires, we also need to mention or link to a doc on what is the current retention period.

Reply:
```suggestion
1. Stop sending data to the tenant
2. Remove tenant from all `GrafanaOrganization` resources (data will be kept until the end of the retention period)
```